internal/cmd/suggest.go

```diff
@@ -77,7 +78,13 @@ or just bad. Please use with discretion.
 			}
 
 			model := tui.NewListModel(ctx, description, repoUser, client)
-			return newProgram(cmd, model).Start()
+			err = newProgram(cmd, model).Start()
+
+			if err == nil {
```

Comment:
Neither this `err` nor model's err (if you expose it through eg `model.Err()`) ever reflect the internal error state. Not sure why.